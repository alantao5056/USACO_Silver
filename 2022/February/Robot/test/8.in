40
-1079 -1054
-165 -20
-95 -232
-856 -776
-610 -935
-401 -110
-221 -506
97 -16
-826 -581
-188 -283
462 269
-768 -457
228 448
17 290
-141 -381
604 622
-742 -512
-109 150
-815 -779
38 -68
992 936
-987 -975
-885 -915
-333 -100
-19 307
-257 -284
218 120
-368 -109
686 499
-558 -382
-413 -361
509 677
963 820
-271 -444
158 128
931 729
-227 -393
20 -312
781 826
551 645
954 672
