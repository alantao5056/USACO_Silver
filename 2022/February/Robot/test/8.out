0
0
0
0
0
1
6
4
51
141
356
892
1878
3519
6093
9372
13078
16462
19052
19948
18888
16254
13014
9201
5975
3619
1921
966
394
160
56
16
5
1
0
0
0
0
0
0
